tests/contrib/BUILD

```diff
@@ -0,0 +1,26 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//contrib:rename_image.bzl", "rename_image")
+
+sh_test(
+    name = "rename_image_test",
+    srcs = ["rename_image_test.sh"],
+    data = [":intermediate_bundle_renamed_image.tar"],
```

Comment:
Is there a way to just use `:renamed_image` target as `data` here? 

If so, I'd suggest to use it; otherwise, let's at least to document it the name very clearly in the rename_image macro. Also, I believe we can come up with a name better than ":intermediate_bundle_%{name}.tar" 😃

Reply:
Changed it to be more clear.